pytest_fixtures/component/taxonomy.py

```diff
@@ -262,3 +280,10 @@ def upgrade_entitlement_manifest():
     upgrade scenarios"""
     manifestor = Manifester(manifest_category=settings.manifest.entitlement)
     return manifestor.get_manifest(), manifestor
+
+
+@pytest.fixture
+def sca_manifest_for_upgrade():
+    """Returns a manifest in sca mode. Used only for upgrade scenarios"""
+    manifestor = Manifester(manifest_category=settings.manifest.golden_ticket)
```

Comment:
`manifestor`?

Reply:
Yes, non-blocking, but I would prefer if you changed the spelling of all instances of this to `manifester`.